warehouse/config/WHTrackEventTableConfig.js

```diff
@@ -13,10 +13,12 @@ const getCloudRecordID = message => {
 };
 
 const rules = {
-  id: (message, options) =>
-    message.type === "track" && options.sourceCategory === "cloud"
+  id: (message, options) => {
+    return message.type === "track" &&
+      sourceCategoriesToUseRecordId.includes(options.sourceCategory)
       ? getCloudRecordID(message)
-      : message.messageId
+      : message.messageId;
+  }
```

Comment:
1. Seems similar code in two places. Can you help me understand WHTrackEventTableConfig and WHTracksTableConfig how are they different?
2. getCloudRecordID method also seems to be exactly same in both files. Is there better way to unify?

Reply:
1. One if for tracks table in customer warehouse. Other if for event table like `add_to_cart` etc. 
2. Moved to single func